src/main/java/com/google/alpollo/model/SongInfo.java

```diff
@@ -23,16 +19,16 @@
   /** TopSalientEntities is a list containing the 10 most important words, given the song context. */
   @Index private List<SongEntity> topSalientEntities;
   @Index private String lyrics;
-  /** After analyzing the entities, we'll have a list of recommended YouTube links. */
-  @Index private List<String> youTubeLinks;
+  /** List of YouTube video IDs based on the most salient entity */
```

Comment:
Problem with the documentation here was in another pr but it's fine. Just drawn your attention.